src/hooks/surveyHooks.ts

```diff
@@ -0,0 +1,2 @@
+// TODO: implement properly
+export const useSurveyLanguages = () => ['de', 'pl'];
```

Comment:
this todo is handled already in a follow up PR or still todo?

Reply:
this is a todo that is still a todo because there is no endpoint yet as far as i know

Reply:
all right